defaults/main.yml

```
@@ -106,6 +106,7 @@ perun_oldgui_logoUrl: 'img/logo.png'
 perun_oldgui_nativeLanguage: 'cs,\u010Cesky,Czech'
 perun_oldgui_language_supported: 'cs'
 perun_oldgui_registrar_findSimilarUsers_disabled: '{{ perun_rpc_registrar_findSimilarUsers_disabled }}'
+perun_oldgui_registrar_hide_proxy: ''
```

The comment in perun-web-gui.properties.j2 says that the value should be a list, not a string.
```suggestion
perun_oldgui_registrar_hide_proxy: []
```